### ISSUE TITLE
luna-devel doesn't install plugins

### ISSUE THREAD

**Issue by mirzal** [theforeman/forklift#950]
As of 91ed38aae72780782368dfe70eafc851a83f7e19 , luna-devel box does not install various Foreman plugins it should, except for Katello.

## Steps to reproduce:
1. `cp vagrant/boxes.d/99-local.yaml.example vagrant/boxes.d/99-local.yaml`
2. `vagrant up centos7-luna-devel`
3. After box is provisioned, ssh into it and `ls ~`

## Actual results

Only two directories are present: `foreman` and `katello`.  
During provisioning, following will be displayed:

```
ok: [centos7-luna-devel] => changed=false 
  ansible_facts:
    foreman_installer_options_joined: ' --disable-system-checks --katello-devel-scl-ruby=rh-ruby25 --katello-devel-admin-password changeme'
```

## Expected results

All plugins are installed

## Additional info

Playbook luna-devel runs `katello_devel` playbook with some additional variables on top. `katello_devel` playbook essentially runs `katello_devel` role. Since b5e90f9e5a510df65e9b8e518efb34d24657aa86 this role specifies variables inside each dependent role, and since fe0964f4fb04b8ffd5fe0b6b0118a9a9fd0f7a20 it doesn't use vars anymore.

It seems that `foreman_installer_options_internal_use_only` set in luna-devel does not overwrite default value in role, and as a result, is not used.

**Comment by san7ket** [comment]
Yea, I installed centos7-luna-devel it installs the plugins but from gems or rpms. The plugins are not build from git repo anymore

**Comment by evgeni** [comment]
sure it started with 91ed38a? because that's a totally unrelated commit :)

**Comment by mirzal** [comment]
@evgeni oh no, I meant that this was the commit I encountered this issue on. It was `master` at the time of writing this issue.

I guess it is caused by b5e90f9e5a510df65e9b8e518efb34d24657aa86 or fe0964f4fb04b8ffd5fe0b6b0118a9a9fd0f7a20 . I tried bisecting this, but it failed a lot while setting up repositories (URLs changed recently?), so I gave up.

But to be honest, it doesn't really matter when it was introduced. I only wanted to provide this information for context.
